microsoft-edge/web-platform/how-to-detect-win11.md

```diff
@@ -75,6 +75,24 @@ navigator.userAgentData.getHighEntropyValues(["platformVersion"])
 
 ```
 
+## Optimizing detection performance
+Since `Sec-CH-UA-Platform-Version` is not a field sent by `Accept-CH` by default, determining this information typically requires a roundtrip every time: 
```

Comment:
I can't quite parse this line. Nothing is sent in `Accept-CH` by default. `Accept-CH` is a response header that a server may optionally add. If the browser sees this header in the response, the next request may include a `Sec-CH-UA-Platform-Version` request header. Could you clarify?

Your clarification might also help with the below explanation of `Critical-CH` since I'm similarly confused there.

Reply:
From Mozilla "The Accept-CH header may be set by a server to specify which client hints headers a client should include in subsequent requests." So it sounds like there are no default fields or values. I will try a short introduction sentence, with some minor wording changes to the rest of this section. Then you and Jenny can let me know if it is an improvement.

Reply:
I think Jenny's edits resolved this.